bin/blade-icons-generate

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env php
+<?php
+
+$currentDir = getcwd();
+
+if (!file_exists($currentDir.'/vendor/autoload.php')) {
+    echo 'Autoload not found.';
+    exit(1);
+}
+
+require $currentDir.'/vendor/autoload.php';
+
+$packageGenerateCommand = require $currentDir.'/config/generation.php';
```

Comment:
I feel like it would probably be good to have a nicer error message than the default failure when a file in a `require` call isn't found. Maybe something like:

```suggestion
$configurationPath = "{$currentDir}/config/generation.php";

if (! file_exists($configurationPath)) {
    echo 'The Blade Icons generator configuration file could not be found.';
    echo "Please create one at: {$configurationPath}";

    exit(1);
}

$packageGenerateCommand = require $configurationPath;
```

Reply:
That's a great idea @owenvoke - updated now.